src/TunePID.cpp

```diff
@@ -132,13 +136,14 @@ int main(int argc, char** argv) {
 	std::getline(std::cin, str);
 	int d_coeff = std::stoi(str);
 
-	can::motor::setMotorMode(serial, can::motor::motormode_t::pid);
 	can::motor::setMotorPIDConstants(serial, p_coeff, i_coeff, d_coeff);
+	can::motor::setMotorPIDMaxPower(serial, 32767);
+	can::motor::setMotorMode(serial, can::motor::motormode_t::pid);
 
 	double period = 8.0;
 
 	std::this_thread::sleep_for(300ms); // wait for encoder position data to arrive
-	int32_t starting_angle = can::motor::getMotorPosition(serial);
+	int32_t starting_angle = can::motor::getMotorPosition(serial).getData();
```

Comment:
This whole thing should probably be a while loop waiting until we get data (with a max timeout) instead of waiting for 300ms.